4-hangman/hangman02.py

```diff
@@ -0,0 +1,13 @@
+import random
+my_list = ["I", "eat", "chicken", "and", "turkey"]
+random_word = str(random.choice(my_list))
+# print(random_word.upper())
+
+word = input("Kindly type one letter only: ")
+index = 0
+while index < len(random_word):
+    if word in random_word[index]:
+        print("YES in {}".format(index))
+    index += 1
+if word not in random_word:
```

Comment:
Your code doesn't account for if I enter a lowercase letter. If I guess 'a', it should be same as 'A'

Reply:
Done